Package.swift

```diff
@@ -105,6 +105,10 @@ let package = Package(
                 "mlx/mlx/distributed/mpi",
                 "mlx/mlx/distributed/ops.cpp",
                 "mlx/mlx/distributed/primitives.cpp",
+
+                // the mlx-c side of distributed
+                "include/mlx/c/distributed.cpp",
+                "include/mlx/c/distributed_group.cpp",
```

Comment:
This is now exposed on the mlx-c layer -- need to discuss how we expose this in swift and specifically how the build will work.